### PR TITLE
[MAP][Ministation]Firefighting boogaloo

### PR DESCRIPTION
**Pull request by kyrahabattoir**
-Added firelocks to departments and central hallway.
-Frequency isolated air alarms in central hallway.
-Relocated a couple of air alarms that where too far from vents.
-Added fire alarms to hallway sections that had none.

### CODE REVIEW

File: _maps/map_files/MiniStation/MiniStation.dmm [hunk 1]
```diff
@@ -12882,6 +12884,14 @@
 	},
 /turf/simulated/floor/plasteel,
 /area/engine/engineering)
+"AR" = (
```
Comment (xxalpha) [inline _maps/map_files/MiniStation/MiniStation.dmm]: This specifically will conflict with your other PR, but not much can be done about that.

**Comment by kyrahabattoir** [comment]
Yeah I noticed the tile indexes where a bit funky.

I'm almost tempted to scratch this PR and do a complete rework of ministation's vents & alarms locations, since many of them are simply poorly placed.

Is mapmerge for TGM just being super weird? should I just 'hold off' on large map changes until that is solved?

**Comment by xxalpha** [comment]
It's not being weird.

**Comment by kyrahabattoir** [comment]
Yeah but if you look at it, it's not cleanly adding the new tiles and removing the old ones such as bR becoming IK for no good reason.

**Comment by xxalpha** [comment]
bR and IK are different.

**Comment by kyrahabattoir** [comment]
Nope, they are identical.

**Comment by xxalpha** [comment]
You added a firedoor and it became different.

**Comment by tkdrg** [comment]
okay i'm not merging this

**Pull request closed without merging**